Wire-iOS/Sources/UserInterface/UserProfile/Devices/UserClientCell.swift

```diff
@@ -117,6 +121,8 @@ class UserClientCell: SeparatorCollectionViewCell {
         let boldAttributes: [NSAttributedString.Key: AnyObject] = [NSAttributedString.Key.font: boldFingerprintFont.monospaced()]
         
         verifiedIconView.image = client.verified ? WireStyleKit.imageOfShieldverified : WireStyleKit.imageOfShieldnotverified
+        verifiedIconView.accessibilityLabel = client.verified ? "verified": "unverified"
```

Comment:
The accessibility label is what is gonna be spoken by VoiceOver, you need to set the identifier with the automation locators, and the label with a localized string.

```suggestion
        verifiedIconView.accessibilityIdentifier = client.verified ? "verified" : "unverified"
        verifiedIconView.accessibilityLabel = client.verified ? "device.verified".localized : "device.not_verified".localized
```